app/services/clients/adminusers.client.js

```diff
@@ -69,7 +68,6 @@ const findServiceBy = function findServiceBy (findOptions, loggingFields = {}) {
 
 module.exports = function (clientOptions = {}) {
   baseUrl = clientOptions.baseUrl || process.env.ADMINUSERS_URL
-  correlationId = clientOptions.correlationId || ''
   return {
     findServiceBy: findServiceBy
```

Comment:
Not really part of your code, but just thought that this can just be `findServiceBy` in ES6 now right? Otherwise looks all good to me 🙂